app/ui/style.css

```diff
@@ -86,41 +92,48 @@ omni-box {
 }
 
 .omni-box-nav-item {
-	border: 1px solid var(--ag-theme-primary);
-	display: flex;
-	flex-direction: row;
-	overflow: hidden;
+  border: 1px solid var(--ag-theme-primary);
+  display: flex;
+  flex-direction: row;
+  overflow: hidden;
   white-space: nowrap;
   text-overflow: ellipsis;
 }
 
 find-menu {
-	border: 2px solid var(--ag-theme-primary);
-	border-top: none;
-	display: flex;
-	flex-direction: row;
-	background: var(--ag-theme-background);
-	color: var(--ag-theme-text);
+  border: var(--border-width) solid var(--ag-theme-primary);
+  border-top: none;
+  display: flex;
+  flex-direction: row;
+  background: var(--ag-theme-background);
+  color: var(--ag-theme-text);
 }
 
 browser-actions {
-	border: 2px solid var(--ag-theme-primary);
-	display: flex;
-	flex-direction: row;
-	background: var(--ag-theme-background);
-	color: var(--ag-theme-text);
-	justify-content: center;
+  height: calc(var(--top-height) - calc(var(--border-width) * 3));
+  border-bottom: var(--border-width) solid var(--ag-theme-primary);
+  border-left: var(--border-width) solid var(--ag-theme-primary);
+  display: flex;
+  flex-direction: row;
+  background: var(--ag-theme-background);
+  color: var(--ag-theme-text);
+  justify-content: center;
   align-items: center;
+  padding: var(--border-width);
 }
-
 .browser-actions-button {
-  padding: 0 0.25em;
+  padding: 0 calc(var(--border-width) * 2);
+  height: calc(100% - calc(var(--border-width) * 2));
+  width: auto;
+} .browser-actions-button>img {
+  height: 100%;
```

Comment:
Of we set this to 1em or something it would size along with the text in the omni box and we wouldn't need to worry about extra calculations, right?

Reply:
In theory